DuckDuckGo/NetworkLeaderboard.swift

```diff
@@ -101,7 +101,8 @@ class NetworkLeaderboard {
         let request: NSFetchRequest<PPTrackerNetwork> = PPTrackerNetwork.fetchRequest()
         request.sortDescriptors = [ NSSortDescriptor(key: "detectedOnCount", ascending: false) ]
         guard let results = try? container.managedObjectContext.fetch(request) else { return [] }
-        return results
+        let pagesVisitedCount = Float(pagesVisited())
+        return results.filter { Float($0.detectedOnCount) / pagesVisitedCount >= 0.01 }
```

Comment:
I haven't got a chance to test it yet, just a question: this looks like this method should return all networks detected, since name indicates so, and this class seems to me like a store interface. I'd say that ignoring occurrences below certain threshold is more of a logic for the Screen that is presenting the data. What do you think?

Reply:
That's a fair point.  "shouldShow" logic in here is arguably just for the UI, so I'll shift this detection level logic there.

Reply:
I've reverted my last commit and gone back to this one.  While this is doing some stuff with persistence it hasn't been strictly abstracted in to a store and was designed to provide data for the UI really. As such 'detected' could also be interpreted as 'detected above a certain level' which is what this is doing. 

This is just quick fix anyway as it's very likely this functionality is going to get ripped out and/or changed "soon". :)